lms/djangoapps/grades/signals/handlers.py

```diff
@@ -158,14 +159,13 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
     Handles the PROBLEM_WEIGHTED_SCORE_CHANGED signal by
     enqueueing a subsection update operation to occur asynchronously.
     """
-    result = recalculate_subsection_grade.apply_async(
+    result = _recalculate_subsection_grade.apply_async(
```

Comment:
Why was this converted to an internal method name when it's used in non-test code outside of its python module?